devops/builds/infrastructure/gateway/gateway.spec

```diff
@@ -6,7 +6,7 @@ import os
 SRCROOT = os.path.normpath(SPECPATH + '/../../../../')
 HOOKSPATH = ['{}/platform/dev/pyinstaller/hooks'.format(SRCROOT)]
 
-a = Analysis(['{}/repomanager.py'.format(SPECPATH)],
+a = Analysis(['{}/gateway.py'.format(SPECPATH)],
```

Comment:
We don't package our apps with pyinstaller anymore so I think all .spec files can simply be removed.